projects/packages/jetpack-mu-wpcom/src/features/launch-button/index.php

```diff
@@ -11,9 +11,29 @@
  * @param WP_Admin_Bar $admin_bar The WordPress admin bar.
  */
 function wpcom_add_launch_button_to_admin_bar( WP_Admin_Bar $admin_bar ) {
+	$current_blog_id = get_current_blog_id();
+
+	if ( function_exists( 'is_graylisted' ) && is_graylisted( $current_blog_id ) ) {
```

Comment:
This seems to be simple site only I guess, not sure how to deal with this kind of conditions.

Reply:
Exactly, only simple sites would be grey listed. I think I've seen precedent to using `function_exists`.